lib/launcher/launcher_test.go

```diff
@@ -168,3 +168,14 @@ func newBrowser() (*launcher.Browser, func()) {
 	b.Context = ctx
 	return b, cancel
 }
+
+func (t T) TestProfileDir() {
```

Comment:
```suggestion
func (t T) ProfileDir() {
```

No need to prefix `Test`